lidder_test.go

```diff
@@ -37,18 +40,70 @@ rules:
       - file_a.go
       - file_b.go`
 
-	d, err := parse([]byte(conf))
+	return parse([]byte(conf))
+}
+
+func (s *Zuite) TestParseConfiguration() {
+	d, err := configFile()
 	require.NoError(s.T(), err)
 
-	require.Equal(s.T(), []string{"^.*\\.go$"}, d.Include)
-	require.Equal(s.T(), []string{"^.*\\bvendor/.*$"}, d.Exclude)
+	require.Equal(s.T(), []string{"^abc/.*\\.go$", "^def/.*\\.go$"}, d.Include)
+	require.Equal(s.T(), []string{"^.*\\bvendor/.*$", "^.*_test\\.go$"}, d.Exclude)
 	require.Equal(s.T(), 1, len(d.Rules))
 	for _, rule := range d.Rules {
 		require.Equal(s.T(), "panic\\(", rule.Pattern)
 		require.Equal(s.T(), []string{"file_a.go", "file_b.go"}, rule.Expected)
 	}
 }
 
+func (s *Zuite) TestShouldCheck() {
+	d, err := configFile()
+	require.NoError(s.T(), err)
+
+	require.False(s.T(), d.shouldCheck("abc/hello_test.go"))
```

Comment:
(Here, you could use `assert.` because it makes sense to continue the test after one failure. Whoever is working on this would get more feedback, all at once.)